powersync-ja/powersync.dart
# feat(attachments): add error handlers

## Description
Add error handlers users can tap into

## Work Done
* Added upload and download error handlers users can use
* Error handler will ignore the attachment if the user opts for this
* Updated the demo to represent this

## How to test
* Create a todo and upload an image
* Delete the app
* Delete the image from local storage
* Reinstall the app
* You should see that the image has been ignored
